PayPalWebPayments/src/main/java/com/paypal/android/paypalwebpayments/PayPalWebCheckoutClient.kt

```diff
@@ -21,7 +22,8 @@ import kotlinx.coroutines.launch
 class PayPalWebCheckoutClient internal constructor(
```

Comment:
Detekt throws on this constructor because it has 7 parameters. Would y'all be opposed to lifting this restriction to 10? I'm sure it's helpful but this lint restriction feels like overkill.

Reply:
I'm down for 10. One of the downsides to composition in large inits and that is OK.

Reply:
I am also down for bumping this to 10.